client/src/pages/MapPage/MapPage.js

```diff
@@ -3,15 +3,31 @@ import './MapPage.scss';
 import MapResults from '../../components/Search/MapResults/MapResults';
 import { connect } from 'react-redux';
 import fetchMonuments from '../../actions/map';
+import {zoom} from "leaflet/src/control/Control.Zoom";
 
 class MapPage extends React.Component {
 
     constructor(props) {
         super(props);
+        this.state = {
+            zoomSize: '5'
+        }
         const { dispatch } = props;
         dispatch(fetchMonuments());
     }
 
+    componentWillMount() {
+        let zoomSize;
+        if(window.innerWidth < 800) {
+            zoomSize = '3';
+        } else if(window.innerWidth < 1600) {
+            zoomSize = '4';
+        } else {
+            zoomSize = '5';
+        }
```

Comment:
This is an unnecessary `else` statement.

I'd just refactor this to be:

`componentWillMount() {
   if (window.innerWidth < 800) {
      this.setState({zoomSize: 3});
   }
   else if (window.innerWidth >= 800 && window.innerWidth < 1600) {
      this.setState({zoomSize: 4});
   }
}`